tests/system-broker/tests/apitests/auth_test.go

```diff
@@ -0,0 +1,147 @@
+package apitests
+
+import (
+	"bytes"
+	"crypto/rsa"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/kyma-incubator/compass/components/connector/pkg/graphql/externalschema"
+	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
+	connectorTestkit "github.com/kyma-incubator/compass/tests/connector-tests/test/testkit"
+	"github.com/kyma-incubator/compass/tests/connector-tests/test/testkit/connector"
+	director "github.com/kyma-incubator/compass/tests/director/gateway-integration"
+	"github.com/pivotal-cf/brokerapi/v7/domain"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTokens(t *testing.T) {
+	// setup
+	runtimeInput := &graphql.RuntimeInput{
+		Name: "test-runtime",
+	}
+
+	logrus.Infof("registering runtime with name: %s, within tenant: %s", runtimeInput.Name, testCtx.Tenant)
+	runtime := director.RegisterRuntimeFromInputWithinTenant(t, testCtx.Context, testCtx.DexGraphqlClient, testCtx.Tenant, runtimeInput)
+	defer director.UnregisterRuntimeWithinTenant(t, testCtx.Context, testCtx.DexGraphqlClient, testCtx.Tenant, runtime.ID)
+
+	logrus.Infof("generating one-time token for runtime with id: %s", runtime.ID)
+	runtimeToken := director.GenerateOneTimeTokenForRuntime(t, testCtx.Context, testCtx.DexGraphqlClient, testCtx.Tenant, runtime.ID)
+	oneTimeToken := &externalschema.Token{Token: runtimeToken.Token}
+
+	logrus.Infof("generation certificate fot runtime with id: %s", runtime.ID)
+	certResult, configuration := connector.GenerateRuntimeCertificate(t, oneTimeToken, testCtx.ConnectorTokenSecuredClient, testCtx.ClientKey)
+	certChain := connectorTestkit.DecodeCertChain(t, certResult.CertificateChain)
+	securedClient := createCertClient(testCtx.ClientKey, certChain...)
+
+	t.Run("Should successfully call catalog endpoint with certificate secured client", func(t *testing.T) {
+		req := createCatalogRequest(t)
+
+		resp, err := securedClient.Do(req)
+		require.NoError(t, err)
+		require.NotNil(t, resp)
+		require.Equal(t, resp.StatusCode, http.StatusOK)
+		body, err := ioutil.ReadAll(resp.Body)
+		require.NoError(t, err)
+
+		require.Contains(t, string(body), "services")
+	})
+
+	t.Run("Should fail calling catalog endpoint without certificate", func(t *testing.T) {
+		req := createCatalogRequest(t)
+
+		client := http.Client{
+			Transport: &http.Transport{
+				TLSClientConfig: &tls.Config{
+					InsecureSkipVerify: true,
+				},
+			},
+		}
+		_, err := client.Do(req)
+		require.Error(t, err)
+		require.Contains(t, err.Error(), "tls: certificate required")
+	})
+
+	t.Run("Should fail calling bind endpoint with revoked cert", func(t *testing.T) {
```

Comment:
Is there something special about the `bind` endpoint compared to the `catalog` one? If not I suggest keeping things simple and consistent and request the `catalog` endpoint throughout the tests to avoid confusion.